src/main/scala/temple/detail/LanguageSpecificDetailBuilder.scala

```diff
@@ -0,0 +1,26 @@
+package temple.detail
+
+import temple.ast.Metadata.ServiceLanguage
+import temple.ast.{Metadata, Templefile}
+import temple.builder.project.ProjectConfig
+import temple.detail.Detail.GoDetail
+
+/** Call out to the user and ask for any required extra input based on their templefile options */
+object LanguageSpecificDetailBuilder {
+
+  private def buildGoDetail(templefile: Templefile, questionAsker: QuestionAsker): Detail = {
+    val packageName = questionAsker.askQuestion(
+      "Yo pal, what is the Go module name you'd like (expected format \"github.com/yeeter69/lockdown\")?",
+    )
```

Comment:
What happens if the user returns the empty string? Surely we want some validation here?